server/modules/elastalert/elastalert.go

```diff
@@ -88,6 +94,17 @@ func (e *ElastAlertEngine) Init(config module.ModuleConfig) error {
 	pkgs := module.GetStringArrayDefault(config, "sigmaRulePackages", []string{"core", "emerging_threats_addon"})
 	e.parseSigmaPackages(pkgs)
 
+	e.reposFolder = module.GetStringDefault(config, "reposFolder", "/opt/sensoroni/sigma/repos")
+	e.rulesRepos, err = module.GetReposDefault(config, "rulesRepos", []*module.RuleRepo{
+		{
+			Repo:    "https://github.com/Security-Onion-Solutions/securityonion-resources",
```

Comment:
By making the securityonion-resources the default/fallback repo, it might not occur to people that choose to specify their own repo in the config that by doing so they are effectively skipping over rules from the SO repo. Would it make sense to move this default repo into the defaults.yaml, so that when they go to this property in configurator, they see it and can choose to replace it or add another to it?

Reply:
Nvm, I see now in the second PR you already list this as a default, so this is a fallback to someone removing the property completely. I think anyway. If they removed the SO repo from the configuration option would it still default to the SO repo here? If so this may be undesired, in the scenario where they explicitly want to stop pulling the SO rules. I think it comes down to how the GetReposDefault will behave when given a property containing an empty list.

Reply:
There's a separate PR with an updated defaults.yaml: https://github.com/Security-Onion-Solutions/securityonion/pull/12640

Reply:
It will only fallback in the case the config doesn't have the `rulesRepos` field present at all. If it's present, even if it's empty, that list is used.